src/main/java/seedu/address/logic/commands/AssignmentStatsCommand.java

```diff
@@ -53,27 +58,28 @@ public CommandResult execute(Model model, CommandHistory history) throws Command
                                         .mapToDouble((marks) -> marks.get(uniqueId).getValue());
         DoubleSummaryStatistics summaryStatistics = markStream.summaryStatistics();
 
-        StringBuilder summary = new StringBuilder(assignment.getName().getValue());
-        summary.append(" Statistics\nSubmissions: ");
-        summary.append(summaryStatistics.getCount());
+        long submissionsCount = summaryStatistics.getCount();
 
-        if (summaryStatistics.getCount() > 0) {
+        double[] quartiles = new double[3];
+        if (submissionsCount > 0) {
 
             markStream = filteredPersonList.stream()
                                             .map(Person::getMarks)
                                             .filter((marks) -> marks.containsKey(uniqueId))
                                             .mapToDouble((marks) -> marks.get(uniqueId).getValue());
             double[] marks = markStream.sorted().toArray();
 
-            double[] quartiles = {0.25, 0.50, 0.75};
-
             if (marks.length == 1) {
                 quartiles[0] = quartiles[1] = quartiles[2] = marks[0];
             } else {
                 double percentile;
                 double position;
                 int index;
 
+                quartiles[0] = 0.25;
```

Comment:
Actually these magic numbers can abstract as variables at start of file?

Reply:
Thanks!